Report/final_report.tex

```diff
@@ -89,186 +91,200 @@
+    We used \verb|typing| to specify the types of inputs that our functions will be taking and some generic types mentioned earlier.
 
-    Lastly, we used ctypes to specifically set an identifier for our application, so the icon will be displayed normally when the application window is active.
+    Lastly, we used \verb|ctypes| to specifically set an identifier for our application when running in Windows platform, so the icon will be displayed normally on the task bar when the application window is active.
 
     \newpage
 
     \section{Instructions}
 
+    Before everything start, please make sure you have a stable connection with the Internet, specifically, GitHub.
+
     \begin{enumerate}
         \item [1]
-            Download the zipped file we provided on MarkUs. The zipped file should contain "algorithms.py, data.py, gui\_main.py, gui\_utils.py, main.py, resource\_manager.py, settings.py, requirements.txt"
+            Download the zipped file we provided on MarkUs. The zipped file should contain ``''\verb|algorithms.py|, \verb|data.py|, \verb|gui_main.py|, \verb|gui_utils.py|, \verb|main.py|, \verb|resource_manager.py|, \verb|settings.py|, and \verb|requirements.txt|.''
 
         \item [2]
-            Unzip the file in a directory with \emph{ONLY} English characters (i.e. Do not place it in a directory that contains non-English characters)
-
+            Unzip the file in an empty directory (It is better to make sure the path to the directory only contains English letters).
         \item [3]
-            Boot up PyCharm and open "requirements.txt" in the directory you just unzipped the files in. Click "Install requirements" to automatically download all the libraries needed
+            Boot up PyCharm and open "requirements.txt" in the directory you just unzipped the files in. Click "Install requirements" to automatically download all the libraries needed.
+
+            Alternatively, you could open the terminal in the directory and run ``pip install -r requirements.txt.''
 
         \item [4]
-            Either use PyCharm or a terminal to run "main.py" and you should be able to see a window popup
+            Either use PyCharm or a terminal to run "main.py" and you should be able to see a window popup.
 
         \item [5]
-            Select a "Sorting algorithm" and click initialize. After a short amount of time, the application should be ready
+            Select a "Sorting algorithm" and click the initialize button. After few seconds, our application should be ready.
 
         \item [6]
-            You should be able to set start and end dates for the plot, pan and zoom on the plot, select countries and plot the result again, and utilize the menu bar for more customizable configurations.
+            You should be able to set the start and end dates for the plot, pan and zoom on the plot, select countries and plot the result again, and utilize the menu bar for more customizable configurations.
```

Comment:
> ...and utilize the menu bar for more customizable configurations.

...and utilize the menu bar for more customizable configurations or even save the plot.